includes/theme-compat/jobs-listing.php

```diff
@@ -0,0 +1,63 @@
+<?php if ( mysqli_num_rows( $jobs_list ) > 0 ) {
```

Comment:
Instead of wrapping the whole file with an IF statement, you can just do:
```suggestion
<?php
if ( mysqli_num_rows( $jobs_list ) < 1 ) {
    echo "0 results";
    return;
}
```

But please do test it after.

Reply:
Also this whole file's formatting is terrible.